docs/DeveloperGuide.md

```diff
@@ -3,36 +3,138 @@
 ## Acknowledgements
 
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+- [CS2113 Course Website](https://nus-cs2113-ay2324s2.github.io/website/index.html)
 
 ## Design & implementation
```

Comment:
Must include an overview of how each classes interact with one another. For instance, include a UML diagram of how UI interacts with exceptions or Parser interacts with UI.